main.py

```diff
@@ -32,9 +32,31 @@ def __init__(self):
     async def on_ready(self):
         print(f"Bot {self.user.name} connected on {len(self.guilds)} servers")
 
+    async def on_command(self, msg):
+        """Log each command submitted. Log message provides information
+        about name, author and arguments of the command.
+        """
+        args = msg.args[2:]
+
+        if args:
+            args_info = f" with args {', '.join(repr(arg) for arg in args)}"
+        else:
+            args_info = ""
+
+        log_msg = f"{msg.command.name} called by {msg.author}{args_info}."
```

Comment:
Put all the human-friendly strings in the `args_info`'s f-string instead of spreading it everywhere (I also one-lined the if, you choose if you want to keep it).
```python
        args_info = ', '.join(repr(arg) for arg in args) if args else ""

        log_msg = f"{msg.command.name} called by {msg.author} with args {args_info}."
```